akka-actor-typed-tests/src/test/resources/logback-doc-test.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <filter class="ch.qos.logback.classic.filter.LevelFilter">
+            <level>DEBUG</level>
+            <onMatch>DENY</onMatch>
+        </filter>
```

Comment:
Use the `<filter class="ch.qos.logback.classic.filter.ThresholdFilter">` instead?

Reply:
Ah, saw the "separate PR" comment now